install/fenrus-install.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# Co-Author: Scorpoon
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+# Source: https://github.com/revenz/Fenrus
+
+
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
```

Comment:
$STD apt-get install -y sudo
$STD apt-get install -y mc

Reply:
Changed it, but could you please explain me why this is needed?

Reply:
`sudo` is precautionary 

All LXC instances created using this repository come pre-installed with Midnight Commander, which is a command-line tool (`mc`) that offers a user-friendly file and directory management interface for the terminal environment.

Base Dependencies created with all LXCs

Reply:
Ok thank you :)